### PR TITLE
Sbogusev/v1.0

### PR DESCRIPTION
**Pull request by emfomenk**

### CODE REVIEW

File: source/elements/oneDNN/source/introduction.rst [hunk 1]
```diff
@@ -137,4 +137,15 @@ a boolean ``allow_empty`` parameter that defaults to ``false`` and that makes
...
+**********
+Namespaces
+**********
+
+All oneDNN functions and classes reside in ``::dnnl`` namespace. The functions
+that accept or return DPC++ objects such as command queues or buffers reside in
+``::dnnl::sycl_interop`` namespace.
+
+Furthermore, oneDNN defines ``::oneapi::dnnl`` namespace, that is an alias for
+the ``::dnnl`` namespace.
```
Comment (sbogusev) [inline source/elements/oneDNN/source/introduction.rst:149]: Do we have to have `::oneapi::dnnl::sycl_interop` too? If so, we may want to mention it explicitly.

File: source/elements/oneDNN/include/dnnl_sycl.hpp [hunk 2]
```diff
@@ -118,12 +118,13 @@ cl::sycl::queue get_queue(const stream &astream);
 ///     does not own the buffer.
 ///
 /// @note
-///     If @p handle is provided, and @p akind is
-///     #dnnl::sycl_interop::memory_kind::buffer, an exception is thrown.
+///     If @p akind is #dnnl::sycl_interop::memory_kind::buffer and @p handle
+///     is different from #DNNL_MEMORY_ALLOCATE or #DNNL_MEMORY_NONE an
```
Comment (sbogusev) [inline source/elements/oneDNN/include/dnnl_sycl.hpp]:
```suggestion
///     If @p akind is #dnnl::sycl_interop::memory_kind::buffer, and @p ahandle
///     is not #DNNL_MEMORY_ALLOCATE or #DNNL_MEMORY_NONE, an
```

File: source/elements/oneDNN/include/dnnl_sycl.hpp [hunk 3]
```diff
@@ -140,11 +141,13 @@ memory make_memory(const memory::desc &adesc, const engine &aengine,
...
```
Comment (sbogusev) [inline source/elements/oneDNN/include/dnnl_sycl.hpp]: Same as above.

File: source/elements/oneDNN/include/dnnl_sycl.hpp (hunk 2 above)
Comment (emfomenk) [reply source/elements/oneDNN/include/dnnl_sycl.hpp]: fixed

**Pull request closed without merging**